### PR TITLE
add files

### PR DESCRIPTION
**Pull request by iYefeng**
This failure is not caused by me.
“Build started
git clone -q https://github.com/OpenRefine/OpenRefine.git C:\projects\openrefine
git fetch -q origin +refs/pull/1139/merge:
git checkout -qf FETCH_HEAD
Specify a project or solution file. The directory does not contain a project or solution file.”

I find that if jython scripts are too long, the status code of the http response is 413 because you put scripts in the url. So I have changed some code again. Please have a check.

fix a bug: the results of runniing jython scripts are wrong when it has Chinses string.

### CODE REVIEW

**Comment by tfmorris** [comment]
Thanks for the pull request. Is there an issue associated with the bug that this fixes? What's the issue number?

**Comment by iYefeng** [comment]
This failure is not caused by me.
“Build started
git clone -q https://github.com/OpenRefine/OpenRefine.git C:\projects\openrefine
git fetch -q origin +refs/pull/1139/merge:
git checkout -qf FETCH_HEAD
Specify a project or solution file. The directory does not contain a project or solution file.”

I find that if jython scripts are too long, the status code of the http response is 413 because you put scripts in the url. So I have changed some code again. Please have a check.

**Comment by jackyq2015** [comment]
Seems it caused by a bug of appveyor:
https://github.com/ogrisel/python-appveyor-demo/issues/6

File: extensions/jython/src/com/google/refine/jython/JythonEvaluable.java [hunk 1]
```diff
@@ -89,7 +90,11 @@ public JythonEvaluable(String s) {
         // indent and create a function out of the code
         String[] lines = s.split("\r\n|\r|\n");
         
-        StringBuffer sb = new StringBuffer(1024);
+        StringBuffer sb = new StringBuffer(102400);
```
Comment (jackyq2015) [inline extensions/jython/src/com/google/refine/jython/JythonEvaluable.java]: initial capacity of 1M is too much? 1k make more sense.

**Comment by wetneb** [comment]
Closing stale pull request

**Pull request closed without merging**